github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1906,3 +1906,46 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: false
+    annotations:
+      fork-per-release: "true"
+    cluster: ibm-prow-jobs
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 1h0m0s
+    labels:
+      preset-bazel-cache: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-podman-in-container-enabled: "true"
+    max_concurrency: 1
+    name: pull-kubevirt-metrics-lint
+    run_if_changed: docs/metrics.md
+    optional: true
+    skip_branches:
+    - release-\d+\.\d+
+    spec:
+      containers:
+      - command:
+        - /usr/local/bin/runner.sh
+        - /bin/sh
+        - -c
+        - |
+          cp /etc/bazel.bazelrc ./ci.bazelrc && make lint-metrics
+        env:
+        - name: PODMAN_IN_CONTAINER_IPV6_ENABLED
+          value: "false"
+        image: quay.io/kubevirtci/golang-legacy:v20220810-a8f2e6c
+        name: ""
+        resources:
+            requests:
+              memory: 8Gi
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: docker-socket
+          mountPath: /var/run/docker.sock
+      volumes:
+      - name: docker-socket
+        hostPath:
+        path: /var/run/docker.sock
```

Comment:
I am not sure if the volume mount for the docker socket should be here. Why do you need this? the podman in podman setup creates a socket in the pod. 

We shouldn't try to mount this from the host.

Reply:
I removed this.